metaswitch/common/mib.py

```diff
@@ -267,3 +278,9 @@ def _get_tokenized_mib_details(mib_file, oid):
                 in_braces = not in_braces
 
     return output
+
+
+class DependencyException(Exception):
+    def __init__(self, message, dependency):

```

Comment:
One more comment: you'll only ever see `MissingDependencyException` in the following contexts:
* This definition (followed by `(Exception)`
* In a `raise` statement
* In a `except` statement

So there's no need to write `Exception` in its name.

How about:
```
class MissingDependency(Exception):
    pass
```